client/lib/products-values/index.js

```diff
@@ -359,6 +364,25 @@ export function getDomain( product ) {
 	return product.meta;
 }
 
+export function getJetpackProductDisplayName( product ) {
+	product = formatProduct( product );
+	assertValidProduct( product );
+
+	return JETPACK_PRODUCT_DISPLAY_NAMES?.[ product?.productSlug ];
+}
+
+export function getJetpackProductTagLine( product ) {
+	product = formatProduct( product );
+	assertValidProduct( product );
+
+	return JETPACK_PRODUCT_TAG_LINES?.[ product?.productSlug ];
+}
+
+export function isProductExpiring( product ) {
+	const expiration = product?.expiryMoment ?? null;
```

Comment:
I am not sure when this would be the case. But if we ever have a product that doesn't expire and then doesn't have a `expiryMoment` we would end up `isProductExpiring()` returning true for that case. 

Is that something we want?

Reply:
I agree with Enej here.

Reply:
I think you're right. In this kind of an edge case we would probably want to check if we have `expiration` and if not then return `false`, i.e.:

```js
export function isProductExpiring( product ) {
	const expiration = product?.expiryMoment;

	if ( ! expiration ) {
		return false;
	}

	return expiration < moment().add( 30, 'days' );
}
```

What do you think?